lib/evss/claims_service.rb

```diff
@@ -5,6 +5,7 @@ module EVSS
   class ClaimsService < BaseService
     API_VERSION = Settings.evss.versions.claims
     BASE_URL = "#{Settings.evss.url}/wss-claims-services-web-#{API_VERSION}/rest"
+    DEFAULT_TIMEOUT = 120 # in seconds
```

Comment:
This might just be my ignorance of the backend, but we already have a timeout for the claims service somewhere, so I'm curious why you're changing it here vs wherever the other timeout value is.

Reply:
@aub Adding to Jeff's question, it looks like there is another DEFAULT_TIMEOUT for EVSS here: https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/evss/configuration.rb#L6

And something else similar here: https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/evss/base_service.rb#L10

Can we either add comments about when each is used or remove any ones that are always overridden to avoid ourselves being confused in the future?